src/app/modules/admin/state/admin.model.ts

```diff
@@ -0,0 +1,18 @@
+export interface Admin {
+    name: string;
+    email: string;
+    avatar: string;
+}
+
+export interface UpdatedProfile {
+    name: string;
+    email: string;
+}
+
+export interface RemoveProfileAvatar {
```

Comment:
please use the following patern for the requests which don't return any data, like deletions or saving smth

export interface NoDataResponce {
    success: boolean;
    message: string;
}
export interface RemoveProfileAvatarResponce extends NoDataResponce  {}